Classes/Emogrifier.php

```diff
@@ -163,7 +163,7 @@ class Emogrifier
         // adjacent sibling
         '/\\s+\\+\\s+/'                            => '/following-sibling::*[1]/self::',
         // descendant
-        '/\\s+/'                                   => '//',
+        '/\\s+(?=.*[^\]]{1}$)/'                    => '//',
```

Comment:
Can you please make the backslash a double backslash to make it clear to the reader that this is no PHP escaping? (And, thinking of that,  I think we need four backslashes here: one set of escaping for PHP, and one set of escaping for regexp.)